css/styles.css

```diff
@@ -509,3 +542,23 @@ button#scroll-to-top img {
     flex-direction: column;
   }
 }
+
+@media (max-width: 839px) {
+  .search-keyword, .search-category {
+    flex-direction: column;
+  }
+  label {
+    margin-bottom: 20px;
+  }
+}
+
+@media (max-width: 877px) {
+  .description {
+    height: 175px;
+  }
+  figure figcaption {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+}
```

Comment:
You'll want a media query that changes the buttons to be stackable in smaller viewports. flex-direction: column works. A good way to test this is to put a custom-sized device in the inspector, say 350px. If you leave them side-by-side then they go off the screen on either side.

Reply:
When I tested this media query at 877px, it did let the buttons stack (instead of being side-by-side, which is their starting positioning). Did you see something different when you were viewing my css changes on your end?
If I misunderstood your comment here, just let me know! :)